src/main/java/frc/robot/RobotContainer.java

```diff
@@ -39,6 +40,6 @@ private void configureButtonBindings() {}
    * @return the command to run in autonomous
    */
   public Command getAutonomousCommand() {
-    return null;
+    return new SimpleAutonCommand(driveBase, Constants.SIMPLE_AUTON_SPEED, Constants.SIMPLE_AUTON_RUNTIME);
```

Comment:
Did you import the constants?

Reply:
RobotContainer.java is in the same package as Constants.java, so you do not need to import `Constants` in this file